src/Microsoft.Azure.WebJobs.Protocols/ParameterLog.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Azure.WebJobs.Host.Protocols
 #if PUBLICPROTOCOL
```

Comment:
The changes you made above removing the PUBLICPROTOCOL stuff from some of the files above, but not here? And you're changing the visibility to public below.